databases/mycli/Portfile

```diff
@@ -1,14 +1,12 @@
-# $Id$
-
 PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        dbcli mycli 1.5.2 v
+github.setup        dbcli mycli 1.8.1 v
 
-python.default_version     27
+python.versions     27 34 35
```

Comment:
BTW I think I did it because of dependencies:
```tcl
depends_build       port:py${python.version}-setuptools
depends_lib-append  port:py${python.version}-click \
                    port:py${python.version}-crypto \
                    port:py${python.version}-pygments \
                    port:py${python.version}-prompt_toolkit \
                    port:py${python.version}-pymysql \
                    port:py${python.version}-sqlparse \
                    port:py${python.version}-configobj
```